base/GoldenSun.ts

```diff
@@ -15,7 +15,7 @@ import {ShopMenu} from "./main_menus/ShopMenu";
 import {InnMenu} from "./main_menus/InnMenu";
 import {ControlManager} from "./utils/ControlManager";
 import {CursorManager} from "./utils/CursorManager";
-import {Gamepad} from "./Gamepad";
+import {Gamepad as XGamepad, Button, CButton} from "./XGamepad";
```

Comment:
Why do we need this `X` before `Gamepad`?

Reply:
The non-X and the alias come from the fact that I was working parallelly with the old Gamepad class and to make sure it was not Phaser.Gamepad.